sensing/ahrs_ekf.cpp

```diff
@@ -472,3 +471,13 @@ bool Ahrs_ekf::update(void)
 
     return task_return;
 }
+
+Mat<7,1>& Ahrs_ekf::x()
+{
+    return x_;
+}
+
+Mat<7,7>& Ahrs_ekf::P()
+{
+    return P_;
+}
```

Comment:
This is only needed because you take non-const references to these matrices in ahrs_ekf_mocap. I would rather remove this as
- it breaks encapsulation
- the const version are already defined in the base class `const Mat<n,1>& Kalman::x() const` and `const Mat<n,n>& Kalman::P() const`